src/components/slider/slider.scss

```diff
@@ -0,0 +1,125 @@
+@import 'default-theme';
+
+$slider-size: 20px !default;
```

Comment:
That's what I was thinking of doing, the only thing that makes me question that is when you have a vertical slider it would then be the width... but that's not in this PR so I'm not sure what's better.